Components/Alerts.html

```diff
@@ -0,0 +1,125 @@
+          </h1></a
+        >
+      </div>
+      <div class="navItems">
+        <a href="./docs.html" class="navDocs">Docs</a>
+        <h3 class="navVersion">Ver. 0.0.1</h3>
+      </div>
+      <div class="burgerIcon" onclick="toggleSidebar()">
+        <ion-icon name="menu-outline"></ion-icon>
+      </div>
+    </nav>
+
+    <!-- MAIN CONTAINER -->
+
+    <div class="mainCont">
+      <!-- SIDEBAR CONTAINER -->
+
+      <div class="sidebarCont hideSidebar">
+        <div class="sidebarList">
+          <a class="sidebarListItem" href="docs.html">Getting Started</a>
+          <hr class="hr-dark mt8" />
+          <a class="sidebarListItem" href="./Typography.html">Typography</a>
+          <a class="sidebarListItem" href="./Buttons.html">Buttons</a>
+          <a class="sidebarListItem" href="./Cards.html">Cards</a>
+          <a class="sidebarListItem" href="./Images.html">Images</a>
+          <a class="sidebarListItem" href="./inputs.html">Inputs</a>
+          <a class="sidebarListItem" href="./lists.html">Lists</a>
+          <a class="sidebarListItem" href="./Navigation.html">Navigation</a>
+          <a class="sidebarListItem" href="./Avatars.html">Avatars</a>
+          <a class="sidebarListItem selected" href="./Alerts.html">Alerts</a>
+          <a class="sidebarListItem" href="./miscellaneous.html"
+            >Miscellaneous</a
+          >
+        </div>
+        <div class="signature-cont">
+          <hr class="hr-dark mt8" />
+          <h4 class="signature-text">
+            Made with 🤎 by
+            <a class="signature-link" href="https://atishayjain.netlify.app/"
+              >Atishay Jain</a
+            >
+          </h4>
+        </div>
+      </div>
+
+      <!-- MAIN CONTENT CONTAINER -->
+
+      <div class="contentCont">
+        <h1 class="heading-l">Alerts</h1>
+        <p class="para-m mt4">Different types of Alerts</p>
+        <hr class="hr-light mt8" />
+
+        <h4 class="heading-s mt10">Examples:</h4>
+
+        <!-- ** Alerts  ** -->
+
+        <div class="exampleCont">
+          <h3 class="exampleText">Alerts</h3>
+
+          <div class="flexCont flex-column">
+            <div class="alert-cont success">
+              <ion-icon
+                name="checkmark-circle-sharp"
+                class="alert-icon"
+              ></ion-icon>
+              <p class="alert-text">Successful</p>
+            </div>
+
+            <div class="alert-cont danger">
+              <ion-icon name="alert-circle-sharp" class="alert-icon"></ion-icon>
+              <p class="alert-text">Danger</p>
+            </div>
+
+            <div class="alert-cont info">
+              <ion-icon name="alert-circle-sharp" class="alert-icon"></ion-icon>
+              <p class="alert-text">Info</p>
+            </div>
+            <div class="alert-cont alert">
+              <ion-icon name="alert-circle-sharp" class="alert-icon"></ion-icon>
+              <p class="alert-text">Warning</p>
```

Comment:
accessibility of alert text(success,info and warning) is not optimal,I would suggest go with colors of alert from material ui